### ISSUE TITLE
kernel_size in qkv_conv

### ISSUE THREAD

**Issue by ruslangrimov** [leaderj1001/Attention-Augmented-Conv2d#11]
Is it right to use self.kernel_size here for kernel_size instead of 1?

https://github.com/leaderj1001/Attention-Augmented-Conv2d/blob/2f6b41f7f1a9ced830e5ec97780da96323bbbc47/AA-Wide-ResNet/attention_augmented_conv.py#L30

**Comment by leaderj1001** [comment]
Thank you for opinion :)
I implemented it in two versions. The first one is implemented in the paper, and the second one is implemented to change the kernel_size.
1) In the paper, they use fixed kernel_size = 1. so, I implemented in "in_paper_attention_augemnted_conv" directory.
https://github.com/leaderj1001/Attention-Augmented-Conv2d/blob/7e070e442c7deb3eb48ddb66e4e84d77bcfa8a51/in_paper_attention_augmented_conv/attention_augmented_conv.py#L22

2) Flexible kernel_size version
- [AA-Wide-ResNet directory](https://github.com/leaderj1001/Attention-Augmented-Conv2d/blob/master/AA-Wide-ResNet/attention_augmented_conv.py)

Thank you !!

**Comment by nguyenvo09** [comment]
I might miss something but in PyTorch,`nn.Conv2d` does not pad the input. It means we need to explicitly use F.padding to pad input. But in your code 
`self.conv_out = nn.Conv2d(self.in_channels, self.out_channels - self.dv, self.kernel_size, stride=stride, padding=self.padding)`
you may pad value 1 or 2 to the tensor input which is undesirable.
